src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -830,13 +826,13 @@ process_includes (RpmOstreeTreeComposeContext  *self,
                   JsonArray *parent_array = json_node_get_array (parent_val);
                   JsonArray *child_array = json_node_get_array (val);
                   JsonArray *new_child = json_array_new ();
-                  guint i, len;
+                  guint len;
```

Comment:
Minor: could also inline this with the assignment.

Reply:
Mmm, there are two assignments right now.